lib/types.ts

```diff
@@ -67,9 +70,28 @@ export const ACTIVITY_TYPES = [
   "pr_merged",
   "pr_collaborated",
 ] as const;
-
+export interface Action {
+  event: string;
+  source: {
+    type: string;
+    issue: {
+      pull_request: boolean;
+      repository: {
+        full_name: string;
+      };
+      user: {
+        login: string;
+      };
+      number: number;
+    };
+  };
+  assignee: {
+    login: string;
+  };
+  created_at: Date;
+}
 export interface Activity {
-  type: (typeof ACTIVITY_TYPES)[number];
+  type: (typeof ACTIVITY_TYPES)[number] | string;
```

Comment:
`| string` defeats the point of having strong types on the left.

```suggestion
  type: (typeof ACTIVITY_TYPES)[number];
```